templates/organization/commissionable_asset/add.html.twig

```diff
@@ -7,6 +7,11 @@
     {{ form_start(form) }}
     {{ form_errors(form) }}
     {{ form_row(form.type, {label: 'Type'}) }}
-    {{ form_row(form.name, {label: 'Nom'}) }}
+    {{ form_row(form.name, {label: 'Identifiant'}) }}
```

Comment:
can you put the labels in the Form Type ? As it, it will be the same for the Edit form